docs/usage/secrets_engines/transit.rst

```diff
@@ -109,7 +109,7 @@ Encrypt Data
 
 	encrypt_data_response = client.secrets.transit.encrypt_data(
 		name='hvac-key',
-		plaintext=base64.urlsafe_b64encode('hi its me hvac').decode('ascii'),
+		plaintext=base64.urlsafe_b64encode('hi its me hvac'.encode()).decode('ascii'),
```

Comment:
I have a vague inkling that this might come down to Python 2.7 versus Python 3.x behavior. If that pans out, perhaps there might be a more universal solution here. I'm going to add doctest coverage to this file to help figure that bit out and add some tweaks to true up things generally (might end up being a ton of tweaks really...).

Reply:
Yeah I think something like https://github.com/hvac/hvac/commit/e96895de9e99e45ec868f2da9e53ce4860595a0d will work out best here. Will merge this PR in and then rebase that branch on top of things.